scripts/ec2/networking.sh

```diff
@@ -29,13 +29,13 @@ readonly EC2_FILES='/var/tmp/ec2'
 
 [[ -d $EC2_FILES ]] || mkdir -p $EC2_FILES
 
-# The version 3.1.2 is currently the recommended version.
-SRIOV_DRIVER='ixgbevf-3.1.2.tar.gz'
+# The version 4.2.1 is currently the recommended version.
```

Comment:
With this driver version, did you see that the support for the `InterruptThrottleRate` module option/parameter is still supported? I am not sure if they still have it in the most recent release - in which case we would need to set the value using ethtool (see: https://sourceforge.net/p/e1000/mailman/message/35774103/; for examples).